packages/webview_flutter/example/lib/main.dart

```diff
@@ -45,6 +46,7 @@ class _WebViewExampleState extends State<WebViewExample> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      backgroundColor: Colors.green,
```

Comment:
Why is this hardcoded as green?

Reply:
It was to show the effect of the transparent background option in the example.
I can remove this change in the example and just keep the "opaque" option to the default value (true).

Reply:
Oh sorry, i missed this was in the example code.  Nevermind.